DataMigration: Adding SQL DB Migration support (#4672)
Azure/azure-cli-extensions: src/datamigration/HISTORY.rst

Release History
===============

0.3.0
++++++

> * [BREAKING CHANGES] Removing parameters provisioing-error and migration-operation-id from commands az datamigration sql-managed-instance create and az datamigration sql-vm create, as they are unnecessary parameters. 
> * [BREAKING CHANGES] Typo correction for parameters --number-of-interation to --number-of-iteration in command az datamigration performance-data-collection
> * [NEW SUBGROUP] az datamigration sql-db - Manage database migrations to SQL DB instance.

Reviewer: ```suggestion
* [BREAKING CHANGE] `az datamigration sql-managed-instance/sql-vm create`: Remove `--provisioing-error` and `--migration-operation-id` as they are unnecessary parameters. 
* [BREAKING CHANGE] `az datamigration performance-data-collection`: Rename `--number-of-interation` to `--number-of-iteration` for typo correction
* [NEW COMMAND GROUP] `az datamigration sql-db`: Manage database migrations to SQL DB instance.
```

Author: Updated.
